### PR TITLE
docs: small edits to the README file.

### PR DESCRIPTION
**Pull request by lwjohnst86**
This is a "stacked PR", where changes are made to a "non-main" branch, so that it is easier to review. Please review and approve #31 first.

### CODE REVIEW

**Review by Aastedet — APPROVED** [review]

**Comment by Aastedet** [comment]
Nice. @lwjohnst86  Can I review/approve the subsequent stacked PRs before this is merged, or do I need to wait for @signekb to approve this one first?

**Comment by lwjohnst86** [comment]
@Aastedet, we need two approving reviewers, so @signekb needs to approve it before it is merged in ☺️

**Pull request merged**
